recipes/wsgiref/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "wsgiref" %}
+{%set version = "0.1.2" %}
+{%set compress_type = "zip" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "c7e610c800957046c04c8014aab8cce8f0b9f0495c8cd349e57c1f7cabf40e79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - wsgiref
+
+about:
+  home: http://cheeseshop.python.org/pypi/wsgiref
+  license: PSF or ZPL
```

Comment:
Nope - all I've got is [what's on pypi](https://pypi.python.org/pypi/wsgiref) - if there's a better source we should definitely use it.